IntegrationTests/tests_01_allocation_counters/test_01_resources/test_empty_3.swift

```diff
@@ -0,0 +1,21 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import X509
+
+func run(identifier: String) {
+    measure(identifier: identifier) {
+        return 0
+    }
+}
```

Comment:
Can we drop the empty tests?

Reply:
remove them

Reply:
the integration test don't like that their are no test files at all. Added one back again. It will not be for long because #97  will replace that test anyway